src/shogun/features/Features.cpp

```diff
@@ -265,6 +265,24 @@ void CFeatures::unset_property(EFeatureProperty p)
 	properties &= (properties | p) ^ p;
 }
 
+CFeatures* CFeatures::view(const SGVector<index_t>& subset)
+{
+	auto feats_view = this->duplicate();
+	feats_view->add_subset(subset.clone());
+	return feats_view;
+}
+
+CFeatures* CFeatures::view(const std::vector<index_t>& subset)
```

Comment:
this doesnt belong into CFeatures imo, this is conversion code between std::vector and SGVector and therefore should sit in SGVector
`features->view(SGVector<float64_t>(my_std_vector))`

Reply:
or even done implicitly (if possible)